app/src/main/java/org/schabi/newpipe/fragments/list/search/SearchFragment.java

```diff
@@ -139,7 +140,7 @@
     @State
     boolean wasSearchFocused = false;
 
-    private Map<Integer, String> menuItemToFilterName;
+    @Nullable private Map<Integer, String> menuItemToFilterName = null;
```

Comment:
Can't this be initialized to an empty map, in order to avoid the null check below?

Reply:
No, it can't, since `.get()` is used on it and that would return a valid value only if it has been initialized.